src/devicecheck.cs

```diff
@@ -20,7 +20,10 @@ namespace DeviceCheck {
 	[Native]
 	public enum DCError : long {
 		UnknownSystemFailure,
-		FeatureUnsupported
+		FeatureUnsupported,
```

Comment:
I wonder how many years it'll take us to add a trailing comma everywhere...

Reply:
likely a few more once sharpie output them correctly ;-)
c.c. @dalexsoto

Reply:
heh maybe that would require some Roslyn 🙈